### DISCUSSION TITLE
Web application

### DISCUSSION THREAD

**Discussion by ncasnap** [fonoster/routr#224] [Q&A] [answered]
Hello,

I see in the roadmap that a web application will be developped. Do you know when this feature will be available ?

Regards,

Nathan

**Comment by psanders** [comment]
Hey @ncasnap, it's hard to tell when the web app will be completed. At the moment, I'm pressing forward to complete the docs and solve bugs. Do you mind sharing your use case? Also, have you considered joining the [Discord](https://discord.gg/4QWgSz4hTC) community?

**Comment by ncasnap** [comment, marked as answer]
Hello @psanders,

Thank you for your feedback. We are looking for an opensource solution with GUI for at least manage devices, create route pattern and sip trunk. 

I'm wondering, Routr is based on asterisk ?

Thank you for your help.

**Reply by psanders** [reply]
Routr is not based on Asterisk. Also, different type of software, with some overlapping use cases. Asterisk is a PBX, Routr is a Programmable SIP Server.
